src/main/java/net/dries007/tfc/common/items/TFCShieldItem.java

```diff
@@ -18,6 +21,9 @@ public TFCShieldItem(Tier tier, Properties builder)
     {
         super(builder.defaultDurability(tier.getUses()));
         this.tier = tier;
+        if (FMLEnvironment.dist.isClient()) {
+            ItemProperties.register(this, new ResourceLocation("blocking"), (stack, world, living, i) -> living != null && living.isUsingItem() && living.getUseItem() == stack ? 1.0f : 0.0f);
+        }
```

Comment:
Prefer grouping this with our other registrations of this type in [ClientEventHandler](https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/5750590698c490b91d4a1526cf9e866b59836264/src/main/java/net/dries007/tfc/client/ClientEventHandler.java#L145-L176)